OpenNefia.Content/Resources/Locale/en_US/Elona/CharaMake/CharaMake.lua

```diff
@@ -0,0 +1,84 @@
+﻿Elona.CharaMake = {
+	WeaponProf = "Proficient in",
```

Comment:
Nitpick: For consistency, prefer not to abbreviate locale keys (`WeaponProficiency`) unless it's obviously impractical to do so.

Reply:
Also maybe `ProficientIn` is better?

Reply:
agreed

Reply:
changed in 76f6762